### PR TITLE
BGDIDIC-2001: Add ch.bafu.amphibienwanderung-verkehrskonflikte

### PR DESCRIPTION
**Pull request by b4l**
Test link: https://mf-chsdi3.dev.bgdi.ch/feature_BGDIDIC-2001/shorten/90253aa4f8

### CODE REVIEW

**Review by faselm — COMMENTED** [review]
Looks good but the legend in French/Italian should be in French or Italian, no?

**Comment by ltclm** [comment]
yes some translations are missing (de and fr are mandatory):

```
$ psql -x -h pg-0.dev.bgdi.ch -d bod_master -c "select * FROM ms_class WHERE fk_layer_name like 'ch.bafu.amphibienwanderung-verkehrskonflikte'" -U www-data
-[ RECORD 1 ]------+---------------------------------------------
ms_class_id        | 6354
pk_ms_class_name   | amphibienwanderung_verkehrskonflikte
fk_layer_name      | ch.bafu.amphibienwanderung-verkehrskonflikte
frm_designation_de | Amphibienwanderungen mit Verkehrskonflikten
frm_designation_fr | Amphibienwanderungen mit Verkehrskonflikten
frm_designation_it |
frm_designation_en |
bgdi_modified      |
bgdi_created       | 2021-03-29 15:37:54.138877
bgdi_modified_by   |
bgdi_created_by    | pgkogis
ms_minscaledenom   |
ms_maxscaledenom   |
bgdi_id            | 5825
frm_designation_rm |
```

**Comment by ltclm** [comment]
@b4l cant see or add the new layer in this testlink.

**Comment by b4l** [comment]
> @b4l cant see or add the new layer in this testlink.

@ltclm this was somewhat lost in transition to wmts. it should be fixed now.

File: chsdi/models/vector/bafu.py [hunk 1]
```diff
@@ -2390,3 +2390,25 @@ class LandesforstinventarSchutzwaldregion(Base, LandesforstinventarRegion, Vecto
...
 
 register(LandesforstinventarSchutzwaldregion.__bodId__, LandesforstinventarSchutzwaldregion)
+
+
+class AmphibienwanderungVerkehrskonflikte(Base, Vector):
+    __tablename__ = 'amphibienwanderung_verkehrskonflikte'
+    __table_args__ = ({'schema': 'fauna', 'autoload': False})
+    __bodId__ = 'ch.bafu.amphibienwanderung-verkehrskonflikte'
+    __template__ = 'templates/htmlpopup/amphibienwanderung-verkehrskonflikte.mako'
+    __queryable_attributes__ = ['nr_datenbank', 'name']
```
Comment (ltclm) [inline chsdi/models/vector/bafu.py:2400]: please dont ask this feature anymore, it wont be supported in our future api anymore.
@mariokeusen is that ok with you?

File: chsdi/models/vector/bafu.py [hunk 2]
```diff
@@ -2390,3 +2390,25 @@ class LandesforstinventarSchutzwaldregion(Base, LandesforstinventarRegion, Vecto
...
+    __label__ = 'name'
+    id = Column('nr_datenbank', Integer, primary_key=True)
```
Comment (ltclm) [inline chsdi/models/vector/bafu.py:2402]: is this attribute delivered with the data and supposed to be stable?

**Review by ltclm — CHANGES_REQUESTED** [review]

File: chsdi/models/vector/bafu.py (hunk 2 above)
Comment (b4l) [reply chsdi/models/vector/bafu.py:2402]: yes, this is supposed to be stable

File: chsdi/models/vector/bafu.py (hunk 1 above)
Comment (b4l) [reply chsdi/models/vector/bafu.py:2400]: how does this relate to the searchable attributes from the DIF?

Comment (ltclm) [reply chsdi/models/vector/bafu.py:2400]: there is no relation between that and the fulltext search in sphinx. this line is for the configuration/activation of the query search tool which will be deprecated soon.

image

searchable attributes in sphinx are only indicating if theres a fulltext search or not.

Comment (b4l) [reply chsdi/models/vector/bafu.py]: sorry, my bad. I removed it in accordance with Mario.

**Review by rebert — APPROVED** [review]

**Pull request merged**